Shoko.Server/Utilities/AVDumpHelper.cs

```diff
@@ -192,56 +192,54 @@ public static string DumpFile(string file)
                 return "Could not find Video File: " + file;
             }
 
-            var filenameArgs = GetFilenameAndArgsForOS(file);
+            var startInfo = GetExecutableAndArgListForOS(file);
 
-            logger.Info($"Dumping File with AVDump: {filenameArgs.Item1} {filenameArgs.Item2}");
-
-            var pProcess = new Process
-            {
-                StartInfo =
-                {
-                    FileName = filenameArgs.Item1,
-                    Arguments = filenameArgs.Item2,
-                    UseShellExecute = false,
-                    WindowStyle = ProcessWindowStyle.Hidden,
-                    RedirectStandardOutput = true,
-                    CreateNoWindow = true
-                }
-            };
+            // Logs only the last argument to AVDump, the filename
+            logger.Info($"Dumping File with AVDump: \"{startInfo.ArgumentList[^1]}\"");
 
+            using var pProcess = new Process();
+            pProcess.StartInfo = startInfo;
```

Comment:
```suggestion
            using var pProcess = new Process { StartInfo = startInfo };
```

Uses object constructor to set the start info at the same time as creation.